src/test/resources/innovimax/mixthem/test021_params-file.txt

```diff
@@ -0,0 +1 @@
+3th 3
```

Comment:
why 3*th* ??? it should be just "3"

Reply:
It is just a free suffix to distinguish test rule runs and that is appended to the name of result file used for the test